geom/src/polygon.rs

```diff
@@ -161,24 +161,14 @@ impl Polygon {
             &self.points
         }
     }
-    pub fn into_points(mut self) -> Vec<Pt2D> {
-        if let Some(mut rings) = self.rings.take() {
-            rings.remove(0).into_points()
-        } else {
-            self.points
-        }
-    }
+    // TODO Switch to get_outer_ring. osm2streets depends on this one, have to juggle git repos
```

Comment:
Some code now lives in the `osm2streets` repo. There's a circular dependency. I could make the branches point at each other, but it's kind of a hassle. So after this PR, I'm going to push fixes to osm2streets, then update it in this repo, and then proceed with cleanup

Reply:
reminder that `patches.crates-io` can make this a little less painful, in that at least you can centralize a lot of the changes - but up to you obviously.